sct-commons/src/test/java/org/lfenergy/compas/sct/commons/scl/SclServiceTest.java

```diff
@@ -1079,13 +1079,13 @@ void analyzeDataGroups_should_return_errors_messages() throws Exception {
         assertThat(sclReport.getSclReportItems()).hasSize(11)
                 .extracting(SclReportItem::getMessage)
                 .containsExactlyInAnyOrder(
-                        "There are too much FCDA for the Client IED IED_NAME1",
-                                "The Client IED IED_NAME1 subscribes to too much GOOSE Control Blocks.",
-                                "The Client IED IED_NAME1 subscribes to too much REPORT Control Blocks.",
-                                "The Client IED IED_NAME1 subscribes to too much SMV Control Blocks.",
-                                "There are too much FCDA for the DataSet DATASET6 for the LDevice LD_INST21 in IED IED_NAME2",
-                                "There are too much FCDA for the DataSet DATASET6 for the LDevice LD_INST22 in IED IED_NAME2",
-                                "There are too much FCDA for the DataSet DATASET5 for the LDevice LD_INST22 in IED IED_NAME2",
+                                "The Client IED IED_NAME1 subscribes to too much FCDA. 9 > (max) 8",
```

Comment:
Update issue or roolback changes

Reply:
I updated the issue in Jira and Github for all messages.